src/sql/parts/query/common/resultSerializer.ts

```diff
@@ -176,6 +177,40 @@ export class ResultSerializer {
 		return fileName;
 	}
 
+	private getResultsFileExtension(saveRequest: ISaveRequest): FileFilter[] {
+		var fileFilters = new Array<FileFilter>();
+		var fileFilter = new FileFilter();
```

Comment:
it looks like you can create a FileFilter using `this.instantiationService.createInstance(FileFilter);` which would allow us to avoid making the interface a class.  We're trying to avoid changes under `vs` to keep changes to underlying VS Code platform minimized.